values.yaml

```
@@ -222,7 +229,23 @@ client:
     # http_proxy: http://localhost:3128,
     # https_proxy: http://localhost:3128,
     # no_proxy: internal.domain.com
+  
+  # hostPath is fullpath to folder on host machine to mount as /consul/data folder. consul agent stores
+  # its configuration in this folder. By default its created as emptyDir: {}. To save data between pod restarts
+  # specify folder name on host machine to be mounted as /consul/data.
+  # Example:
+  # hostPath: '/var/consul-data'
+  hostPath: null
+
+  # hostPath is fullpath to folder on host machine to mount as /consul/data folder. consul agent stores
+  # its configuration in this folder. By default its created as emptyDir: {}. To save data between pod restarts
+  # specify folder name on host machine to be mounted as /consul/data.
+  # Example:
+  # hostPath: '/var/consul-data'
+  hostPath: null
 
+  # Deafult DnsPolicy is not default in kube. Specify your prefered DnsPolicy here.
+  DnsPolicy: "Default"
```

```suggestion
  dnsPolicy: null
```
All keys must start with lowercase